input/fsh/pdqm.fsh

```diff
@@ -39,6 +39,118 @@ Description: "The patient demographics and identifier information which can be p
+except the input resource SHALL be an instance of the [PDQm Patient Profile for $match Input](StructureDefinition-CHPDQmMatchInput.html).
+
+Note that the only REQUIRED parameter is the Patient Resource. When only the Patient is supplied, it can be POSTed directly to the $match endpoint
+without being wrapped in a Parameters Resource, as long as it conforms to the [PDQm Patient Profile for $match Input](StructureDefinition-CHPDQmMatchInput.html).
+"""
+* parameter[resource].resource only CHPDQmMatchInput
+
+// https://github.com/IHE/ITI.PDQm/blob/main/input/fsh/PDQmMatch.fsh
+Profile: CHPDQmMatchParametersOut
+Parent: Bundle
+Id: ch-pdqm-matchparametersout
+Title: "CH PDQm Match Output Bundle Profile"
+Description: "A profile on the Query Patient Resource Response message for ITI-119"
+* type = #searchset (exactly)
+* total 1..
+* entry ^slicing.discriminator.type = #type
+* entry ^slicing.discriminator.path = "resource"
+* entry ^slicing.rules = #open
+* entry.fullUrl 1..
+* entry contains Patient 0..5 and OperationOutcome 0..*
+* entry[Patient] ^short = "Patient"
+* entry[Patient].resource 1..
+* entry[Patient].resource only ch-pdqm-patient
+* entry[Patient].search 1..
+* entry[Patient].search.mode 1..
+* entry[Patient].search.mode = #match
+* entry[Patient].search.score 1..
+* entry[Patient].search.extension contains http://hl7.org/fhir/StructureDefinition/match-grade named MatchGrade 1..1
+* entry[OperationOutcome] ^short = "OperationOutcome"
+* entry[OperationOutcome].resource 1..
+* entry[OperationOutcome].resource only OperationOutcome
+// * entry[OperationOutcome].resource ^type.code = "OperationOutcome"
+// * entry[OperationOutcome].resource ^type.profile = Canonical(OperationOutcome)
+// * entry[OperationOutcome].resource only ch-pdqm-moreattributesrequested
+
+// https://github.com/IHE/ITI.PDQm/blob/main/input/fsh/PDQmMatch.fsh
+Profile: CHPDQmMatchInput
+Parent: https://profiles.ihe.net/ITI/PDQm/StructureDefinition/IHE.PDQm.MatchInputPatient
+Title: "CH PDQm Patient Profile for $match Input"
+Description: """
+The PDQm Patient Profile for $match Input SHALL be provided as input to the ITI-119 transaction.
+- While it is not REQUIRED that the input to $match be a valid FHIR instance, it is RECOMMENDED to supply as many elements as possible to facilitate matching.
+- modifierExtension and implicitRules SHALL not be specified.
+- The patient-mothersMaidenName extension is available to hold the mother's maiden name
+"""
+* name MS                                     // LivingSubjectName
+* gender MS                                   // LivingSubjectAdministrativeGender
+* birthDate MS                                // LivingSubjectBirthTime
+* address MS                                  // PatientAddress
+* identifier MS                               // LivingSubjectId
+* extension[MothersMaidenName] MS             // MothersMaidenName
```

Comment:
we should also add extension birthName instead of MothersMaidenName

Reply:
Fixed. I sliced `name`, as it's done in _CHPDQmPatient_